src/lib/sortPoints.test.js

```diff
@@ -0,0 +1,48 @@
+const sortPoints = require("./sortPoints")
+
+test ('an input of one point returns same point', function() {
```

Comment:
Your two tests seem to do a good job at testing inputs for the sortPoints() function. So far I think your two tests seem very valid and are a great start. It would also be a good idea to test points with different y values and constant x values and test points with different points with all different values for x and y. So far so good :)